oxenss/storage/database.cpp

```diff
@@ -817,8 +819,9 @@ std::vector<std::string> Database::update_expiry(
             impl->db,
             multi_in_query(
                     "UPDATE messages SET expiry = ?"
-                    " WHERE owner = (SELECT id FROM owners WHERE pubkey = ? AND type = ?)"
-                    " AND hash IN ("sv,  // ?,?,?,...,?
+                    " WHERE owner = (SELECT id FROM owners WHERE pubkey = ? AND type = ?)"s +
```

Comment:
it would have slightly more stylistic consistency to have the space before the `WHERE owner =` on the previous line so the update and where line up.

Reply:
never mind i understand it now.